src/video-recorder/process.js

```diff
@@ -143,23 +150,25 @@ export default class VideoRecorder extends AsyncEmitter {
 
         await delay(FFMPEG_START_DELAY);
     }
+    async dispose () {
```

Comment:
We separate different methods and functions with an empty line:
```suggestion
    }

    async dispose () {
```